projects/site2019/src/assets/data/speakers.json

```diff
@@ -297,5 +297,116 @@
             "icon": "",
             "name": "Maggie Goreng Tambah Telur Mata"
         }
+    },
+    {
+        "id": "hongtat-yew",
+        "name": "HongTat Yew",
```

Comment:
is your name really together or separate? 
- It's ok If you like to put it this way, or
- Hong Tat, Yew
- Hong Tat
- Yew Hong Tat

Reply:
it's actually yew hong tat. but people tend to call me hong, yew or yew tat. so i will make it as a word and they know hongtat is first name and yew is last name.